v7/latex/latex/plugins/latex_formula_image_renderer.py

```diff
@@ -89,6 +89,7 @@ def __init__(self):
         super(LatexImageFormulaRenderer, self).__init__()
         self.__formula_scale = 1.25
```

Comment:
That, and they may mess up in some situations where you never expect it. This probably won’t happen in a Nikola plugin that doesn’t get inherited though.